spec/NewTwitchApi/Webhooks/WebhooksSubscriptionApiSpec.php

```diff
@@ -80,51 +80,42 @@ function it_subscribes_to_user_follows(Client $guzzleClient)
         $this->subscribeToUserFollows('12345', '54321', 1, 'https://redirect.url', 'bearer-token', 100);
     }
 
-    function it_subscribes_to_a_user_without_bearer(Client $guzzleClient)
```

Comment:
FYI - Instead of removing this, it could have been changed to expect an exception/error/etc.